vignettes/performance_analysis.Rmd

```diff
@@ -0,0 +1,323 @@
+
+In May 2020, the at-large voting system was struck down, and a ward system with a new set of districting maps was required. In a ward system, voters elect a representative for their own geographically compact ward. Two maps were proposed: one by the plaintiffs (New York Civil Liberties Union, NYCLU) and the defendants (ERSD). Ideally, the map should allow sufficient representation for the minority aggregate population (in this case, Black and Hispanic/Latino voters).
+
+In this case study, we'll focus on the defendant map. We'll demonstrate how a performance analysis reveals that simply using CVAP to assess the minority constituency may overestimate the number of seats won by minority supported candidates.
+
+### Ward Map of East Ramapo School District
+
+Let's take a look at a map proposed by the defendants, ERSD. The district map is composed of nine wards. To assess representation, we could examine the CVAP (the number of people who can vote) by race across the wards. Thus, let's take a look at the fraction of CVAP voters that are in the minority aggregate, by ward:
+
+```{r load_data}
+# Load the map
+data("ersd_maps")
+```
+
+
+```{r cvap_map, fig.align = "center", fig.height=6, fig.width=7.2, message=FALSE, warning=FALSE}
+# Plot the map, using a fill that depends on Citizen Voting Age Population (CVAP)
+options(repr.plot.width = 7.2, repr.plot.height = 6)
+cvap_map <- ggplot() +
+  geom_sf(data = ersd_maps, aes(fill = MIN_AGG_FRAC)) +
+  geom_sf_label(data = ersd_maps, aes(label = WARD), size = 5) +
+  scale_fill_continuous(limits = c(0, 1)) +
+  xlab("Latitude") +
+  ylab("Longitude") +
+  theme_bw(base_size = 10) +
+  theme(
+    axis.title.x = element_text(size = 15, face = "bold", margin = margin(t = 5)),
+    axis.title.y = element_text(size = 15, face = "bold", margin = margin(r = 5)),
+    legend.key.width = unit(0.4, "cm"),
+    legend.key.height = unit(1, "cm")
+  ) +
+  guides(fill = guide_legend(
+    title = "Fraction\nMinority",
+    title.position = "top",
+    title.size = 10
+  ))
+show(cvap_map)
+```
+
+Examining this map reveals that, according to CVAP, four wards (1-4) have potential for minority voters to elect a representative of their choice (they have a plurality or majority). However, due to turnout differences across racial groups, this does not imply that the minority voters would have actually turned out sufficiently enough to elect four representatives. In other words, this map may not have "performed" well enough to guarantee representation, due to turnout.
+
```

Comment:
Might want to add a transition sentence here like "Next, we'll walk through how to conduct a performance analysis and test this hypothesis".